.travis.yml

```diff
@@ -36,6 +36,13 @@ matrix:
       osx_image: xcode8.3 # [`xcode8.3` is Xcode 8.3.3 on OS X 10.12](https://docs.travis-ci.com/user/reference/osx#OS-X-Version)
       language: generic
       env: JOB=package
+    - if: tag IS present # additional JOB to run the release acceptance test for releases only
```

Comment:
Please comment on why these additional tests are run on separate job rather than existing job that builds package (and that uploads that on tag), and whether this is meant to be affected once mining speed becomes configurable and the fastmining profile gets removed. This will help when restructuring tests.